system/lib/libc/musl/src/time/strftime.c

```diff
@@ -226,7 +226,7 @@ size_t __strftime_l(char *restrict s, size_t n, const char *restrict f, const st
 			s[l] = 0;
 			return l;
 		}
-		if (*f != '%') {
+		if (*f != '%' || !f[1]) {
```

Comment:
Can you mention this in system/lib/libc/README.md and put the change inside `#ifdef __EMSCRIPTEN__`?